bot/cogs/clickup.py

```diff
@@ -112,6 +119,8 @@ def __init__(self, bot: AutoShardedBot):
                     status = f"{task['status']['status'].title()}"
 
                     lines.append(f"{id_fragment} ({status})\n\u00BB {task['name']}")
+
+                log.warning(f"{ctx.author} requested a list of ClickUp tasks. Returning list.")
```

Comment:
Incorrect level, this should be an info/debug log, no warnings or errors have occured to reach this code.